docker-compose.prod.yml

```diff
@@ -0,0 +1,134 @@
+#mysql container
+db:
+    restart: always
+    image: mysql:5.6
+    ports:
+        - "3306:3306"
+    environment:
+        MYSQL_ROOT_PASSWORD: changeme
+        #the values below should match the credentials used in prod.env
+        MYSQL_USER: nbsap
+        MYSQL_PASSWORD: nbsap
+        MYSQL_DATABASE: nbsap
+
+#per country container
+ch:
+    image:nbsap_web
+    volumes:
+        - ./instance_ch/:/nbsap/instance
+    ports:
+        - "8000:7012"
+    links:
+        - db
+    env_file:
+        - ./env_files/ch.env
```

Comment:
Sorry, but I still don't get it. Which are the essential files you are referring to? Per instance env files? They will contain sensitive information, so why should they be pushed to the repo? The dev.env stands as an example on how they should look like.